### PR TITLE
JU-319 Adding the initial ecs-image-build directory.

### PR DESCRIPTION
**Pull request by harishkm2k**

### CODE REVIEW

**Comment by ch-code-analysis** [comment]
CI: No IaC security warnings found

File: terraform/groups/ecs-service/locals.tf [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+  global_prefix               = "global-${var.environment}"
+  service_name                = "registered-email-address-api"
+  container_port              = "8080"
+  eric_port                   = "10000"
+  docker_repo                 = "registered-email-address-api"
+  kms_alias                   = "alias/${var.aws_profile}/environment-services-kms"
+  lb_listener_rule_priority   = 86
+  lb_listener_paths           = [ "/transactions/*/registered-email-address*",
+                                  "/private/transactions/*/registered-email-address/filings",
+                                  "/registered-email-address/healthcheck",
```
Comment (ivana-companieshouse) [inline terraform/groups/ecs-service/locals.tf:14]: Do we need to expose the health-check?

Comment (harishkm2k) [reply terraform/groups/ecs-service/locals.tf]: Removed it now.

**Comment by ch-code-analysis** [comment]
CI: No IaC security warnings found

**Review by ivana-companieshouse — APPROVED** [review]

**Pull request merged**